src/Altinn.Broker.Core/Domain/FileReceipt.cs

```diff
@@ -4,7 +4,7 @@ namespace Altinn.Broker.Core.Domain;
 
 public class FileReceipt {
     public Guid FileId { get; set; }
-    public Actor Actor { get; set; }
+    public Actor? Actor { get; set; }
```

Comment:
Are there file receipts without an Actor?

Reply:
presumably not, for the most part I've made these changes to reduce the amount of warnings.

Reply:
We can merge in as-is I think, but we should strive to get around those warnings without changing the semantics of the domain.